### PR TITLE
removed h1 form titles from form pages

### PR DESCRIPTION
**Pull request by bowerscodes-COP**
###Description
Removed H1 Headings from Form Headings

https://support.cop.homeoffice.gov.uk/browse/COP-11023

### CODE REVIEW

File: client/src/pages/tasks/TaskPage.jsx [hunk 1]
```diff
@@ -173,7 +173,6 @@ const TaskPage = ({ taskId }) => {
         document.title = `${processDefinition.name} | ${t('header.service-name')}`;
         sectionComponent = (
           <>
```
Comment (aliktb) [inline client/src/pages/tasks/TaskPage.jsx:175]: do we need this fragment here?

**Review by aliktb — COMMENTED** [review]
Looks good, just a minor comment

File: client/src/pages/tasks/TaskPage.jsx (hunk 1 above)
Comment (bowerscodes-COP) [reply client/src/pages/tasks/TaskPage.jsx]: Good spot! Fixed

**Review by aliktb — APPROVED** [review]
Approved

**Pull request closed without merging**